### PR TITLE
Add core engineering group meeting nodes.

### PR DESCRIPTION
**Pull request by realvictorprm**
Can someone have a check whether this looks OK?

First time I write meeting notes 😖 

Signed-off-by: realvictorprm <mueller.vpr@gmail.com>

### CODE REVIEW

**Comment by smoothdeveloper** [comment]
Looks great, seems there was a dense meeting happening, thanks for making notes public!

File: _posts/2018-11-19-open-core-engineering-group-meeting.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+This online session 18 people joined to discuss the term "Core Engineering".
+
+Questions could be posted during the meeting with prefix "QUESTION:" and as it was an open session also comments could be added with the "COMMENT:" prefix.
+
+The discussion started with a reference to the [google-document](https://docs.google.com/document/d/1wAAR0v1dglXXQThqaMN_iNHi5xSH1boAlP3q-_4kSWI/edit?usp=sharing) to explain the reason why the discussion is happening. Afterwards Don Syme stated the goals for the meeting being to come up with a list of actionable improvements that the community can make to ease development in the core F# repositories and document those in the github repository Microsoft/visualfsharp or elsewhere.
+
+The particular proposed solutions / taks named Proposed Core Engineering Task (PCET) can be found at the end of the document.
+
+The list of questions and ideas of the google document where worked through in the whole meeting starting with:
```
Comment (dsyme) [inline _posts/2018-11-19-open-core-engineering-group-meeting.md]: Mention the authors of the doc here, thanks

File: _posts/2018-11-19-open-core-engineering-group-meeting.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- Discussions about features take months up to years without decision.
+- Some features with many votes are open since years and no decision has been taken on them too
+- Some RFC’s for accepted features are such incomplete that it’s not possible to start implementing them
+- Expected timeframe for an accepted feature with started implementation isn’t available
+
+Don Syme states he's fine with the current approach of handling language suggestions speaking of not closing suggestions rather instead labeling suggestions with "proably-not". However he agrees that the RFC list needs some love and should be cleaned of such which may be better closed and or rebooted. To address this issue the PCET 1 was created where importantly a RFC may be rebooted if it doesn't get a comment or change within a couple of months. Regular meetings to clean up RFC's have been disagreed to aswell voting about whether a RFC should be closed, the process itself prefered be asnychronous only.
```
Comment (dsyme) [inline _posts/2018-11-19-open-core-engineering-group-meeting.md]: Typo "proably"

File: _posts/2018-11-19-open-core-engineering-group-meeting.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+Afterwards the current situation that Don Syme is only deciding whether a language suggestion is accepted took place with the result that in future there won't be a different process to the current including the refreshed statement that Don Syme is the official F# Language Design Lead supported by the FSSF and Microsoft. Carefully also was added that the FSSF in principle is allowed to change this situation through voting however this scenario appears very unlikely. But Don Syme would like to have in future a more active oversight of the design process considering to report every 6 months to the FSSF board.
+
+Furthermore a discussion takes place about whether the design governance or process increase the difficulty of working in the compiler (with different opinions).
+
+For language design certain statements are now fully official:
+- The decision on TCs is clear: we won't put a fully-fledged version of TCs into F# while there is any chance that the corresponding feature will be put into C# (quote Don Syme)
```
Comment (dsyme) [inline _posts/2018-11-19-open-core-engineering-group-meeting.md]: TC --> Type Classes

File: _posts/2018-11-19-open-core-engineering-group-meeting.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- Dave Thomas
+- Oskar Gewalli
+- Thomas Boby
+- Bryant Huang
+- Gerard
+- Elliott V. Brown
+- Diego Esmerio
+- Victor Peter Rouven Müller
+
+### Proposed core engineering tasks:
```
Comment (dsyme) [inline _posts/2018-11-19-open-core-engineering-group-meeting.md]: We should think about how we track these.  We can discuss this next Monday

File: _posts/2018-11-19-open-core-engineering-group-meeting.md (hunk 1 above)
Comment (realvictorprm) [reply _posts/2018-11-19-open-core-engineering-group-meeting.md]: OK, wasn't sure whether it's appropiate to mention me as author as I'm already writing the meeting notes.

File: _posts/2018-11-19-open-core-engineering-group-meeting.md (hunk 4 above)
Comment (realvictorprm) [reply _posts/2018-11-19-open-core-engineering-group-meeting.md]: I agree on this.

File: _posts/2018-11-19-open-core-engineering-group-meeting.md (hunk 1 above)
Comment (dsyme) [reply _posts/2018-11-19-open-core-engineering-group-meeting.md]: It's best to write it as if there were two separate voices (two "hats") - one as the meeting note taker, one as yourself

**Comment by realvictorprm** [comment]
Thanks :)

**Pull request merged**